src/module/rules/rule-element/mark-target/prompt.ts

```diff
@@ -0,0 +1,73 @@
+import { TokenPF2e } from "@module/canvas";
+import { TokenDocumentPF2e } from "@scene";
+import { PredicatePF2e } from "@system/predication";
+
+/** Prompt the user to target a token */
+class MarkTargetPrompt {
+    prompt: string;
+
+    requirements: TargetRequirements | null;
+
+    #target?: TokenDocumentPF2e | null;
+
+    #resolve?: (value: TokenDocumentPF2e | null) => void;
+
+    constructor(params: PromptParameters) {
+        this.prompt = params.prompt ?? "Target a token by mouse-hovering one and pressing T. Press Escape to cancel.";
```

Comment:
I think you forgot to localize this before making the PR?

Reply:
oh yeah, I had it sitting in a stash for a month and forgot where I left it 😅